lab1.py

```diff
@@ -0,0 +1,59 @@
+
+
+# Task1
+num = int(input("Enter a number: "))
+
+
+flag = False
+
+# prime numbers are greater than 1
+if num > 1:
+
+    for i in range(2, num):
+        if (num % i) == 0:
+
+            flag = True
+
+            break
+
+
+if flag:
+    print(num, "is not a prime number")
+else:
+    print(num, "is a prime number")
+
+# Task2
+
+
+def show_student(name, GPA=0):
+    print("name: "+name)
+    if GPA:
+        print("GPA: "+GPA)
+    else:
+        print(0)
+
+
+show_student("sarah", "4.90")
+show_student("sarah")
+
+# Task3
+
+subjects_grades = {'Physics': 90, 'Math': 100, 'history': 70}
```

Comment:
Good 👍🏽
or you can simply use the built-in functions (`min` and `max`) this way: 
```
min(subjects_grades, key= subjects_grades.get)
max(subjects_grades, key= subjects_grades.get)
```